templates/python/api.mustache

```diff
@@ -0,0 +1,260 @@
+# coding: utf-8
+
+{{>partial_header}}
+
+from __future__ import absolute_import
+
+import re  # noqa: F401
+
+# python 2 and python 3 compatibility library
+import six
+
+from {{packageName}}.api_client import ApiClient
+from {{packageName}}.exceptions import (  # noqa: F401
+    ApiTypeError,
+    ApiValueError
+)
+
+
+{{#operations}}
+class {{classname}}(object):
+    """NOTE: This class is auto generated by OpenAPI Generator
+    Ref: https://openapi-generator.tech
+
+    Do not edit the class manually.
+    """
+
+    def __init__(self, api_client=None):
+        if api_client is None:
+            api_client = ApiClient()
+        self.api_client = api_client
+{{#operation}}
+
+    def {{operationId}}(self, {{#requiredParams}}{{^vendorExtensions.x-isDomain}}{{paramName}}, {{/vendorExtensions.x-isDomain}}{{/requiredParams}}{{#requiredParams}}{{#vendorExtensions.x-isDomain}}{{paramName}}="default",{{/vendorExtensions.x-isDomain}}{{/requiredParams}} **kwargs):  # noqa: E501
```

Comment:
Original template: https://github.com/OpenAPITools/openapi-generator/blob/v4.3.1/modules/openapi-generator/src/main/resources/python/api.mustache

This is the one line I changed in the template. It now checks for `x-isDomain` on the parameter and if present it puts the param as a key-word argument with the default of `"default"`. Without this change `domain` would always be a required positional argument and break all of our current usage of the bindings in the tests. I did a quick check and didn't see any spots in our tests that use the `_with_http_info` method variant found below, so I left that untouched and its signature will now require `domain` if the feature is enabled. I can also change it to behave the same if we think it'll be more beneficial.

Reply:
We do use `_with_http_info` in the pulp_file rbac tests, so I'll also modify that line as well.